PublicMapping/districtbuilder
# Build out Organization Admin page and allow projects to be set as featured

## Overview

- Adds a new Organization Admin Screen
- Organization admin screen is protected by a guard, and if any user other than an Organization's designated admin attempts to access it they see our generic 404 error page
- Add in new endpoint `api/project_templates/:slug`for fetching all Project Templates associated with an Organization (and Projects associated with them)
- Add `admin` field to Organization
- Add in new endpoint for toggling featured projects `api/projects/:id/toggleFeatured` that is restricted to admin users
- Updates to `update-organization` to save admin for an organization

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/110140701-5ccc5100-7da2-11eb-9dc4-da89dd42117d.png)


### Notes

I copied the fields from @jfrankl 's wireframe here, but there are a lot of fields from Project Template and Project that are not being displayed here that might be nice to see.

There are a few things built here where the implementation seems a little clunky currently - I will add comments to highlight for futher review


## Testing Instructions

- `./scripts/update`
- Add your Admin user ID to your organization YAML file
- `./scripts/manage update-organization {/YOUR_YAML_FILE.yaml}`
- Log in with your Admin user and attempt to go to `http://localhost:3003/o/{SLUG}/admin`
- Expect: See page
- Attempt to toggle a project as featured
- Expect: Count of featured projects updates at bottom

- Log in with a non-Admin user and attempt to go to `http://localhost:3003/o/{SLUG}/admin`
- Expect: 404 error page displayed


Closes #566
